client/blocks/site/style.scss

```
@@ -141,6 +141,19 @@
 	margin-top: 2px;
 }
 
+.site__title-with-chevron-icon {
+	display: flex;
+
+	.site__title {
+		position: relative;
+		flex: 1;
+	}
+	.site__title-chevron-icon {
```

[NITPICK, NON-BLOCKER] Can we add an extra line space here?
```suggestion
	}
	
	.site__title-chevron-icon {
```